common/djangoapps/student/management/commands/create_user.py

```diff
@@ -34,7 +34,7 @@ def add_arguments(self, parser):
         parser.add_argument('-u', '--username',
                             metavar='USERNAME',
                             help='Username, defaults to "user" in the email')
-        parser.add_argument('-n', '--name',
+        parser.add_argument('-n', '--proper_name',
```

Comment:
I wonder if this command is used by any automated process that will be broken by this parameter name change? Maybe @edx/devops knows?

Reply:
If so they would have been broken for the last ~month, but definitely worth checking.

Reply:
The two things I'm aware of that fiddle with users in an automated way are Devstack's provision and app-permissions and the manage_edxapp_users_and_groups.yml playbook, both of which use the manage_user command.  If it is likely devstack should be using create_user instead of manage_user, I'll defer that to you folks.

This doesn't mean there isn't something else hiding, but those are the two big ones I know ow.